triangleRender.js

```diff
@@ -0,0 +1,47 @@
+let isRotated = false;
+
+function render() {

```

Comment:
this can be split in 3 parts smaller functions, so they can be reused
input validation, 
triangle text generation
and text attachment to the dom.